docs/internal-developers/testing/releases/1005.md

```diff
@@ -0,0 +1,18 @@
+# Testing notes and ZIP for release 10.0.5
+
+Zip file for testing: [woocommerce-gutenberg-products-block.zip](https://github.com/woocommerce/woocommerce-blocks/files/11555954/woocommerce-gutenberg-products-block.zip)
+
+## WooCommerce Core
+
+### Fix Mini-Cart block conflict with Page Optimize and Product Bundles. [(9586)](https://github.com/woocommerce/woocommerce-blocks/pull/9586)
+
+1. Make sure you have WC core 7.7.
+2. Enable a block theme.
+3. Add the Mini-Cart block to the header of your store.
+4. Install the Page Optimize and Product Bundles plugins (no need to change anything in their configuration).
+5. Go to a page in the frontend that doesn't have any blocks besides the Mini-Cart you added to the header.
+6. Open the Mini-Cart and verify there is no JS error:
+
+Before | After
+--- | ---
+![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/307b07c5-0c59-4d04-9599-8cc38691ead9) | ![imatge](https://github.com/woocommerce/woocommerce-blocks/assets/3616980/108f21c1-a658-4441-9bad-910ec701bb36)
```

Comment:
`imatge` typo?

Reply:
My system that defaults to Catalan to set the image alt text. :sweat_smile: Should be fixed now. It's not the most helpful `alt` text, but I think we usually don't set alt texts in testing instructions.